include/conversation.php

```diff
@@ -721,6 +721,7 @@ function conversation(App $a, $items, $mode, $update, $preview = false, $order =
 					continue;
 				}
 
+				$arr = ['item' => $item];
```

Comment:
This fixes the warning but doesn't fix the wrong `Addon::callHooks` call. See https://github.com/friendica/friendica/issues/5335#issuecomment-403286569

Reply:
I'm not totally sure if we have to remove the addon call or not. I would suggest checking it at some later time, since there is still much work to do.

Reply:
No problem, we'll leave the other issue open.